mltrace/entities/base_component.py

```diff
@@ -36,23 +36,28 @@ def __init__(
         self._tags = tags
         self._beforeTests = beforeTests
         self._afterTests = afterTests
+        self._history = history.History(name)
```

Comment:
Make self._history a property by adding the method:

```python
@property
def history(self) -> history.History:
  return self._history
```

That way in your custom component you can call `self.history` in the `afterRun` instead of `self._history`